changelog/bugfixes/2022-01-13-policycoreutils-fix-root-path.md

```diff
@@ -0,0 +1 @@
+- `policycoreutils` root path to `/var/lib/selinux` ([flatcar-linux/Flatcar#596](https://github.com/flatcar-linux/Flatcar/issues/596))
```

Comment:
```suggestion
- Fixed leak of SELinux policy store to the root filesystem top directory due to wrong store path in `policycoreutils` instead of `/var/lib/selinux` ([flatcar-linux/Flatcar#596](https://github.com/flatcar-linux/Flatcar/issues/596))
```